front/src/views/sign-up/sign-up.view.js

```diff
@@ -0,0 +1,47 @@
+import { MmtTextInput } from '@components';
+export default {
+  components: {
+    MmtTextInput
+  },
+  data () {
+    return {
+      valid: true,
+      email: '',
+      firstPass: '',
+      secondPass: '',
+      showPass: false,
+      existingEmail: false,
+      rules: {
+        isEmpty: v => !!v || 'Заполните поле.',
+        minPassLen: v => v.length >= 8 || 'Не менее 8 символов',
+        validEmail: v => /.+@.+\..+/.test(v) || 'Не действительный e-mail',
+        samePass: () => this.firstPass === this.secondPass || 'Пароли должны совпадать',
+        existingEmail: () => !this.existingEmail || 'Этот e-mail уже занят. Попробуйте другой.'
+      }
+    };
+  },
+  methods: {
+    signUp () {
+      if (this.valid) {
```

Comment:
you should pass to an action email and password and body should be created in the action